### DISCUSSION TITLE
Kata 1

### DISCUSSION THREAD

**Discussion by JMOLINAG24** [LaunchX-InnovaccionVirtual/CursoIntroPython#57] [General]
Alguien me podría apoyar con la siguiente pregunta, ¿se debe hacer en un nuevo documento?

**Comment by DianaMagallanes** [comment]
Puedes solo ejecutar los códigos de la kata pero lo mejor seria que hicieras tu propio documento de acuerdo a los pasos del archivo de guía.

**Reply by carlos-gmx3** [reply]
Confirmo, anotando en markdowns tratando de explicar con tus propias palabras le entiendes mucho mejor, además, permitete jugar con el código, experimenta con  la información de los modulos para que veas como se comporta el código en una u otra situación

**Comment by PlayerOneV** [comment]
Si descargaste el repositorio puedes simplemente ejecutar las celdas que están en modulo0Katas.ipynb. En mi caso hice un archivo nuevo y escribí todo para familiarizarme con las celdas y markdown.

**Reply by JMOLINAG24** [reply]
Muchas gracias compañero, una pregunta mas, el la kata 1 como crearon el archivo .py?

**Reply by PlayerOneV** [reply]
Click derecho sobre la carpeta en la que quieres que se guarde, seleccionas new file y le cambias el nombre y le pones la extensión .py.
La extensión .py es para un archivo de Python puro si lo que quieres es crear un notebook de Jupyter le tienes que poner la extensión .ipynb .
Si descargas el repositorio puedes utilizar los archivos que dicen kata para resolver los ejercicios ya que tienen las celdas listas para que escribas código.
![1](https://user-images.githubusercontent.com/75816087/153320456-6ab77c01-97ef-451f-a286-e95719a059d3.PNG)

**Reply by CuauPillado** [reply]
@PlayerOneV Perdón si mi pregunta es algo tonta, ¿pero cómo descargas el repositorio? ¿Primero debes hacer Fork y ya que lo tengo en mi profile puedo descargar los archivos? Quise hacerlo como viene en el taller de Github pero no encontré la opción y supuse que es por ser un documento interno de la organización, y no quise picarle a nada ni hacer Fork para no moverle a los documentos. Sin embargo, todo lo ando haciendo manual en los notebooks.

**Reply by PlayerOneV** [reply]
Te posicionas en el repositorio pero no de la organización si no de Fernanda Ochoa.
![image](https://user-images.githubusercontent.com/75816087/153323101-b62168e8-c7e1-4aca-bbd9-f776d39fce1a.png)
Presionas el boton code y seleccionas download ZIP.
![image](https://user-images.githubusercontent.com/75816087/153323200-ad2929d6-ddfb-43b1-8f93-20cd9a0f2d8f.png)
Lo descomprimes en la carpeta que quieras y en Visual Studio Code vas a File (arriba a la izq.) y eliges "Add Folder to Workspace..." buscas la carpeta de descomprimiste y listo.

Por lo que tengo entendido si se puede usar Fork pero yo no lo he usado antes así que para evitar complicarme con algo que aun no explican descargue el ZIP en lo que explican en donde y como se van a subir los ejercicios.

**Reply by JMOLINAG24** [reply]
Amigoooos!! después de hacer el programa, no puedo continuar.
¿Alguien me pude apoyar?

**Reply by PlayerOneV** [reply]
@JMOLINAG24 comparte aqui unas capturas de pantalla para que pueda saber cual es tu duda.

**Reply by JMOLINAG24** [reply]
Creé el programa 
![PHYTON](https://user-images.githubusercontent.com/99219604/153325668-52856012-9121-40ce-9bfc-eaee1bd99931.jpg)
 Y lo quiero ejecutar pero me manda un error. (lo ejecute en una notebook nueva)
![image](https://user-images.githubusercontent.com/99219604/153325895-7967216e-2497-41f1-89f3-11b429f00f09.png)

¿me podrías confirmar si es correcto?

**Reply by PlayerOneV** [reply]
Mira en la primera imagen le pusiste dos veces la extensión .ipynb, quítale una y te debería aparecer un el triangulo a la izq. de la celda como en la segunda imagen, ese triangulo de play sirve para ejecutar el código que hayas escrito en esa celda.

En la segunda imagen te sale error por que estas tratando ejecutar un archivo en una celda de código. Creo que lo que intentabas hacer era ejecutar un programa por consola y para eso te tienes que posicionar en la ruta en que esta guardado y después poner el nombre del programa pero eso no es algo que se haga para ejecutar un programa cuando tienes Visual Studio Code.

Estoy pendiente por si necesitas mas ayuda.

**Reply by JMOLINAG24** [reply]
Muchas gracias amigo, si pude hacerlo, era mas fácil de lo que pensaba.
De verdad muchas gracias por el apoyo :) me sirvieron mucho tus pantallas

**Comment by CuauPillado** [comment]
Yo vengo desde cero en programación y al igual que PlayerOneV, hice archivos nuevos para familiarizarme con los notebooks. Este módulo es muy sencillo y te va a ayudar bastante a entenderle si lo haces de esa forma :)

**Reply by JMOLINAG24** [reply]
Yei, yo igual ando desde cero, aplicare esa. 
Disculpa tu en la katha 1, ¿como creaste el archivo .py?

**Reply by LeonardoCelisH** [reply]
Si estas haciendo uso del notebook no es necesario crear un .py, se ejecuta en el mismo entorno.

**Reply by CuauPillado** [reply]
No te aseguro que esté en lo correcto (sigo aprendiendo) pero en la carpeta de Python, en la barra de tareas, viene un programa llamado IDLE, es el que te permite crear archivos python (así lo encontré en internet, jejeje), pero tanto las katas como la documentación las estoy haciendo en los notebook que estoy creando, para poder ejecutar los códigos.
